composer.json

```diff
@@ -12,10 +12,10 @@
     }
   ],
   "require": {
-    "php-http/httplug-bundle": "^1.7",
-    "php-http/curl-client": "^1.7",
+    "php-http/httplug-bundle": "^2.0",
+    "php-http/curl-client": "^2.0",
     "php-http/message": "^1.0",
```

Comment:
do we need to bump the version of message too?

Reply:
No, not really. Or not until we add the BC layer that Mark started on.

Reply:
cool